### PR TITLE
Feature/df 643 switching company

### PR DESCRIPTION
**Pull request by ganqqwerty**

### CODE REVIEW

File: alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    value: accountability.companyId
+  } as SelectableOption;
+}
+
+@Component({
+  selector: 'alv-accountability-switcher',
+  templateUrl: './accountability-switcher.component.html',
+  styleUrls: ['./accountability-switcher.component.scss']
+})
+export class AccountabilitySwitcherComponent implements OnInit {
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.ts]: what happens when the users selects/clicks/switches an accountability?

File: alv-portal-ui/src/app/dashboard/dashboard-header/dashboard-header.component.html [hunk 2]
```diff
@@ -1,11 +1,6 @@
 <div class="d-flex align-items-center">
-
   <div class="flex-grow-1">
     <div>{{'portal.dashboard.header.welcome.text' | translate}},</div>
-    <div class="greeting">{{user.firstName}} {{user.lastName}}</div>
```
Comment (pavax) [inline alv-portal-ui/src/app/dashboard/dashboard-header/dashboard-header.component.html]: why is that gone now?

File: alv-portal-ui/src/app/shared/layout/user-menu/user-menu.component.html [hunk 3]
```diff
@@ -31,6 +31,15 @@
                [icon]="['fas', 'hand-point-right']"></fa-icon>
       {{'global.menu.account.registration.incomplete' | translate}}
     </button>
+    <ng-container *ngIf="currentAccountability$ | async as currentAccountability">
+      <small class="ml-2">Companies</small>
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/layout/user-menu/user-menu.component.html]: I18n

File: alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
+::ng-deep alv-select > .form-group > .form-label-group > select  {
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.scss]: this doens't work on chrome for me

File: alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.ts [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+      select(getAccountabilities),
+      filter(Boolean),
+      map((accountabilities: Accountability[]) => accountabilities.map(mapAccountabilityToSelectOption))
+    );
+
+    this.store.pipe(
+      select(getCurrentAccountability),
+      filter(Boolean),
+      map(mapAccountabilityToSelectOption)
+    ).subscribe(currentOption => this.accontabilityFormControl.patchValue(currentOption.value));
```
Comment (pavax) [inline alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.ts]: unsubscribe -> using takeUntil component is destroyed (use the AbstractSubscriber parent)

File: alv-portal-ui/src/app/dashboard/dashboard-header/dashboard-header.component.html [hunk 6]
```diff
@@ -1,11 +1,6 @@
...
-  </div>
-
-  <div *ngIf="company">
-    {{company.companyName}}
   </div>
+  <alv-accountability-switcher></alv-accountability-switcher>
```
Comment (pavax) [inline alv-portal-ui/src/app/dashboard/dashboard-header/dashboard-header.component.html]: the accountability switch MUST NOT be shown for anyone that has no accountability (currently it is shown for everyone) ->I'd suggest that the component itselfs does not render anything if there are no accountabilities

Comment (pavax) [reply alv-portal-ui/src/app/dashboard/dashboard-header/dashboard-header.component.html]: image
image

File: alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.ts (hunk 1 above)
Comment (ganqqwerty) [reply alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.ts]: done

File: alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.scss [hunk 7]
```diff
@@ -0,0 +1,6 @@
+:host {
+  ::ng-deep alv-select > .form-group > .form-label-group > select  {
+    padding-right: 2em;
```
Comment (birom) [inline alv-portal-ui/src/app/shared/layout/accountability-switcher/accountability-switcher.component.scss]: This does not seem to be working on chrome:
selection_162

**Pull request merged**